src/economy.cpp

```diff
@@ -103,38 +103,65 @@ static PriceMultipliers _price_base_multiplier;
 
 /**
  * Calculate the value of the company. That is the value of all
- * assets (vehicles, stations, etc) and money minus the loan,
+ * assets (vehicles, stations, shares) and money minus the loan,
  * except when including_loan is \c false which is useful when
  * we want to calculate the value for bankruptcy.
- * @param c              the company to get the value of.
+ * @param c the company to get the value of.
  * @param including_loan include the loan in the company value.
  * @return the value of the company.
  */
 Money CalculateCompanyValue(const Company *c, bool including_loan)
+{
+	Money companySharesValue = 0;
+
+	int sharesOwned = 0;
+
+	for (const Company* co : Company::Iterate()) {
+
+		sharesOwned = 0;
+
+		for (int i = 0; i < 4; i++){
+			if (co->share_owners[i] == c->index) {
+				sharesOwned++;
+			}
+		}
+
+		if (sharesOwned > 0)
+		{
```

Comment:
The opening curly brace for if statements stays on the same line.

```suggestion
		if (sharesOwned > 0) {
```

Reply:
I made the requested change.